src/views/Dashboard.vue

```diff
@@ -9,7 +9,7 @@
 
     <v-container id="dashboard-container" class="view-container">
       <article id="dashboard-article">
-        <v-row>
+        <v-row class="mt-n9">
```

Comment:
This was simpler than removing the top margin on the sections in each column (some of which are conditional) (and they use common CSS, so it would be weird to override that IMO).